stripe/src/main/java/com/stripe/android/model/Token.java

```diff
@@ -134,4 +149,71 @@ public Card getCard() {
     public BankAccount getBankAccount() {
         return mBankAccount;
     }
+
+    @Nullable
+    public static Token fromString(@Nullable String jsonString) {
+        try {
+            JSONObject tokenObject = new JSONObject(jsonString);
+            return fromJson(tokenObject);
+        } catch (JSONException exception) {
+            return null;
+        }
+    }
+
+    @Nullable
+    public static Token fromJson(@Nullable JSONObject jsonObject) {
+        try {
```

Comment:
Token is a lot older than all the other models, and was brought over mostly intact from stripe-java. I actually could switch this one around not to need try/catch -- i should probably do that with this PR, come to think of it.